src/vs/platform/terminal/common/environmentVariableCollection.ts

```diff
@@ -13,13 +13,17 @@ const mutatorTypeToLabelMap: Map<EnvironmentVariableMutatorType, string> = new M
 	[EnvironmentVariableMutatorType.Prepend, 'PREPEND'],
 	[EnvironmentVariableMutatorType.Replace, 'REPLACE']
 ]);
+// TODO: key was --> 'VSCODE_BASH_ACTIVATE:::file:///Users/anthonykim/Desktop/Skeleton'
+const PYTHON_ACTIVATION_VARS_PATTERN = /^VSCODE_(PWSH|ZSH|BASH)_ACTIVATE/;
```

Comment:
missing the trailing $, key was weird showing ::: after the expected activate key:

`'VSCODE_BASH_ACTIVATE:::file:///Users/anthonykim/Desktop/Skeleton'`

Reply:
I don't remember that but I guess that's how it works. Kartik may have added that part 🤔